nft_ingester/src/program_transformers/mod.rs

```diff
@@ -69,7 +69,7 @@ impl ProgramTransformer {
         &self,
         tx: &'a TransactionInfo<'a>,
     ) -> Result<(), IngesterError> {
-        println!("Handling Transaction: {:?}", tx.signature());
+        info!("Handling Transaction: {:?}", tx.signature());
```

Comment:
this needs to be info?

Reply:
Quite useful for debugging yes. Volume isn't too high at the moment.